pkg/topo/get.go

```diff
@@ -127,6 +160,103 @@ func runGetKindCommand(cmd *cobra.Command, args []string) error {
 	return runGetCommand(cmd, args, topoapi.Object_KIND)
 }
 
+func runGetObjectsCommand(cmd *cobra.Command, args []string) error {
```

Comment:
I'm thinking that we probably ought to break this method up... into three.. this outer one and then two new auxiliary methods, each containing one of the two main branches here.